stable/spinnaker/values.yaml

```diff
@@ -7,6 +7,8 @@ halyard:
   # Set to false to disable persistence data volume for halyard
   persistence:
     enabled: true
+  # Provide additional parameters to halyard deploy apply command
+  additionalInstallParameters: ""
```

Comment:
Could you make this into a list and have the helm command deploy it neatly with new lines please?

It would help making it more readable.

Reply:
I've updated it as your suggestion.